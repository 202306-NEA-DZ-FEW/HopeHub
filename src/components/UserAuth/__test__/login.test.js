import renderer from "react-test-renderer";

import Login from "../login";

it("renders correctly", () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
});
