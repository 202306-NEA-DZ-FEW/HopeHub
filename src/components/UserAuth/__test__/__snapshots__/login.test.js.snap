// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="pt-2 md:pt-40  font-poppins flex flex-col items-start justify-center w-full "
>
  <h2
    className=" font-extrabold text-4xl text-NeutralBlack my-8"
  >
    Log In
  </h2>
  <form
    className="text-4xl p-6 w-full bg-NeutralWhite dark:bg-Dark_Primary rounded-md shadow-md flex flex-col gap-4"
    onSubmit={[Function]}
  >
    <input
      className="input input-bordered w-full bg-NeutralWhite text-NeutralBlack mb-2 mt-1   "
      onChange={[Function]}
      placeholder="Enter your e-mail"
      type="email"
      value=""
    />
    <input
      className="input input-bordered w-full bg-NeutralWhite text-NeutralBlack mb-2 "
      onChange={[Function]}
      placeholder="enter password"
      type="password"
      value=""
    />
    <div
      className="flex flex-row justify-between items-center gap-4"
    >
      <button
        className="btn border-none font-poppins font-regular bg-Accent text-NeutralBlack dark:text-NeutralWhite dark:bg-Dark_Accent dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8] hover:text-NeutralWhite hover:scale-105 duration-500  flex-1 "
        type="submit"
      >
        Log In
         
      </button>
      <button
        className="btn border-none  font-poppins font-regular bg-Accent text-NeutralBlack dark:text-NeutralWhite dark:bg-Dark_Accent dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8] hover:text-NeutralWhite hover:scale-105 duration-500  flex-1"
        onClick={[Function]}
        type=""
      >
        Sign Up
         
      </button>
    </div>
    <span
      className="text-Accent font-semibold text-lg text-right cursor-pointer m mt-1 mb-2 focus:border-b-2 border-Accent"
      onClick={[Function]}
    >
      forgot
    </span>
  </form>
</div>
`;
